src/packages/default-feature-toggle-store/default-feature-toogle-store.test.ts

```diff
@@ -0,0 +1,15 @@
+import defaultFeatureToggleStore from './';
+
+describe('Default Feature Toogle Store', () => {
+  it('should Features.CONSOLE_LISTENER return false', () => {
+    expect(defaultFeatureToggleStore['perry::feature::consolelistener']).toBe(false)
+  })
+
+  it('should Features.DOCUMENT_CLICK_LISTENER return false', () => {
+    expect(defaultFeatureToggleStore['perry::feature::documentclicklistener']).toBe(false)
+  })
+
+  it('should Features.WINDOW_ERROR_LISTENER return false', () => {
+    expect(defaultFeatureToggleStore['perry::feature::windowerrorlistener']).toBe(false)
+  })
+})
```

Comment:
maybe here is a good place to make use of the snapshots feature from jest, since this only validates the Feature Toggle Store default configuration.

what do you think? @vitormargis @guilhermelimak